### PR TITLE
Perform interactive mode check using try catch

### PR DESCRIPTION
**Pull request by ragavsathish**
Fix for #9

### CODE REVIEW

**Comment by ragavsathish** [comment]
@dpo Could you review the changes and kindly provide some feedback.

File: +logging/logging.m [hunk 1]
```diff
@@ -154,6 +154,8 @@ function critical(self, message)
       else
         self.logLevel_ = logging.logging.OFF;
       end
+      swingError = javachk('swing');
+      self.using_terminal = (~ isempty(swingError) && strcmp(swingError.identifier, 'MATLAB:javachk:thisFeatureNotAvailable')) || ~desktop('-inuse');
```
Comment (dpo) [inline +logging/logging.m]: Thanks. It would be great if you could add a comment to explain the test.

**Comment by dpo** [comment]
Thank you. Could you remove the merge commit though?

**Comment by ragavsathish** [comment]
I removed the merge commit while merging could you squash your changes and merge the pull request, please.

**Comment by dpo** [comment]
Thank you.

**Pull request closed without merging**